src/MenuGraphr.ts

```diff
@@ -89,6 +94,7 @@ export class MenuGraphr implements IMenuGraphr {
         this.replacerKey = settings.replacerKey || "%%%%%%%";
         this.sounds = settings.sounds || {};
         this.modifierScope = settings.modifierScope || this;
+        this.listMenuPositions = {};
```

Comment:
So I need to instantiate ItemsHoldr somewhere - is this supposed to happen in the constructor? Not really sure how to go about that. And will I need to make an items.ts file that will only contain 'listMenuPositions`?